Framework/Algorithms/src/PDDetermineCharacterizations.cpp

```diff
@@ -57,18 +57,8 @@ const std::string PDDetermineCharacterizations::summary() const {
  * @return The list of expected column names
  */
 std::vector<std::string> getColumnNames() {
-  std::vector<std::string> names;
-  names.push_back("frequency");  // double
-  names.push_back("wavelength"); // double
-  names.push_back("bank");       // integer
-  names.push_back("container");  // string
-  names.push_back("vanadium");   // string
-  names.push_back("empty");      // string
-  names.push_back("d_min");      // string
-  names.push_back("d_max");      // string
-  names.push_back("tof_min");    // double
-  names.push_back("tof_max");    // double
-  return names;
+  return {"frequency", "wavelength", "bank",  "container", "vanadium",
+          "empty",     "d_min",      "d_max", "tof_min",   "tof_max"};
```

Comment:
It seemed like a strange place to document types of the values for various columns. I'll move that information into the doxygen comment where it should be a bit more accessible.